packages/plugin-core/src/showcase/FeatureShowcaseToaster.ts

```diff
@@ -0,0 +1,106 @@
+import { VSCodeEvents } from "@dendronhq/common-all";
+import { MetadataService, ShowcaseEntry } from "@dendronhq/engine-server";
+import _ from "lodash";
+import * as vscode from "vscode";
+import { AnalyticsUtils } from "../utils/analytics";
+import { ALL_FEATURE_SHOWCASES } from "./AllFeatureShowcases";
+import {
+  DisplayLocation,
+  FeatureShowcaseUserResponse,
+  IFeatureShowcaseMessage,
+} from "./IFeatureShowcaseMessage";
+
+/**
+ * Class to showcase certain features of Dendron as a toast message
+ */
+export class FeatureShowcaseToaster {
```

Comment:
does it make sense to have a toast AND a tip of the day? should we use the toast when the tip isn't visible and vice versa?

Reply:
I think for now yes - I'd prefer to show in both locations until we collect more data on how effective the display locations are.